src/views/Project/index.jsx

```diff
@@ -124,6 +126,22 @@ const tagReposMapping = repositories =>
   spinner: {
     marginTop: 3 * theme.spacing.unit,
   },
+  seeMoreButton: {
+    position: 'absolute',
+    left: 3 * theme.spacing.unit,
+    top: 160,
+  },
+  transparentToWhiteBackground: {
+    background: 'linear-gradient(to bottom, transparent 0%, white 50%)',
+    right: 0,
+    left: 0,
+    top: 120,
+    height: 80,
+    position: 'absolute',
+  },
```

Comment:
The "See More" button is too close to the end of the paragraph. Let's keep the same distance as before:

<img width="657" alt="screen shot 2019-02-20 at 9 13 02 pm" src="https://user-images.githubusercontent.com/3766511/53138511-86680600-3554-11e9-9ba5-35fc944a35da.png">

Before:

<img width="1327" alt="screen shot 2019-02-20 at 9 12 55 pm" src="https://user-images.githubusercontent.com/3766511/53138538-ab5c7900-3554-11e9-8ef3-0b5919a92458.png">

If this makes it hard to implement, then I'd be fine removing the fade out effect.

Reply:
Let me try this out